website/docs/docs/use-dbt-semantic-layer/gsheets.md

```diff
@@ -1,11 +1,10 @@
 ---
-title: "Google Sheets (beta)"
+title: "Google Sheets"
 description: "Integrate with Google Sheets to query your metrics in a spreadsheet."
 tags: [Semantic Layer]
-sidebar_label: "Google Sheets (Preview)"
+sidebar_label: "Google Sheets"
 ---
 
-# Google Sheets <Lifecycle status='preview'/>
 
 :::info Preview functionality
 Google Sheets integration with the dbt Semantic Layer is a [Preview](/docs/dbt-versions/product-lifecycles#dbt-cloud) feature.
```

Comment:
we should prob remove this callout too right @rpourzand ?

Reply:
Yep! good call